src/fidesops/core/config.py

```diff
@@ -41,6 +41,7 @@ class ExecutionSettings(FidesSettings):
     MASKING_STRICT: bool = True
     CELERY_BROKER_URL: str = "redis://:testpassword@redis:6379/1"
     CELERY_RESULT_BACKEND: str = "redis://:testpassword@redis:6379/1"
+    WORKER_ENABLED: bool = True
```

Comment:
Is there a particular reason why we default our users to `True` but ourselves use `false`? - https://github.com/ethyca/fidesops/pull/770/files#diff-b9a882b5033bf73236083b237701d90db30d7405950cbba9cfee37df20bdb191R37

Reply:
They're generally deploying Fidesops in production, so should be encouraged to use a robust setup. We're using Fidesops in a development context and might not need to focus on the execution of privacy requests always, and it's mainly important when execution is happening.